docs-client/build.gradle

```diff
@@ -35,7 +38,8 @@ task buildWeb(type: NpmTask) {
                  'webpack.config.ts').withPathSensitivity(PathSensitivity.RELATIVE)
 
     outputs.dir('build/web')
-    outputs.cacheIf { true }
+    // cache the result from linux builds only to preserve previous behavior
+    outputs.cacheIf { current() == LINUX }
```

Comment:
Note: What this does is only enable caching this task for linux builds, which means that it won't be cacheable for any build not executed on linux.